roles/k8s_best_practices_certsuite/tasks/pre-run.yml

```diff
@@ -28,21 +28,21 @@
       else certsuite execution will fail.
   ignore_errors: true
 
-# Use https://quay.io/repository/testnetworkfunction/k8s-best-practices-certsuite?tab=tags for
```

Comment:
I would change all this code with the following, just to make sure we don't have any conflicts because the new repo is used from v5.2.3, and here the condition is for v5.2.2:

```
# Use https://quay.io/repository/redhat-best-practices-for-k8s/certsuite?tab=tags for
# newer certsuite releases, starting from v5.2.3.
# Note https://quay.io/repository/testnetworkfunction/cnf-certification-test?tab=tags is deprecated
# and its support will be removed at some point.
- name: Set kbpc_image_name
  ansible.builtin.set_fact:
    kbpc_image_name: "{{ (kbpc_version is version('v5.2.3', '>=') or kbpc_version == 'HEAD') | ternary('certsuite', 'cnf-certification-test') }}"
```